src/main/java/entity/Item.java

```diff
@@ -0,0 +1,50 @@
+package entity;
+
+public abstract class Item {
+    private final String name; // is this final??? // the name of the entity.Item
```

Comment:
yea I imagine it should be final. Since equipments and collectibles shouldn't really ever have their name changed